packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import {
+  Button,
+  FormCard,
+  LabeledTextField,
+  Form,
+} from '@asap-hub/react-components';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+
+import { useState } from 'react';
+import { documentTypeMapper } from './CreateOutputPage';
+
+type OutputFormType = {
+  createOutput: (
+    payload: gp2Model.OutputPostRequest,
+  ) => Promise<gp2Model.OutputResponse>;
+  documentType: gp2Routing.OutputDocumentTypeParameter;
+};
+const OutputForm: React.FC<OutputFormType> = ({
+  createOutput,
+  documentType,
+}) => {
+  const [title, setTitle] = useState('');
+  const currentPayload = {
+    title,
+    documentType: documentTypeMapper[documentType],
+  };
+
+  return (
+    <Form onSave={() => createOutput(currentPayload)} dirty={title !== ''}>
+      {({ isSaving, onSave }) => (
+        <div>
```

Comment:
Do we need this div? <>?

Reply:
currently I'm not doing any styling, but I will be needed in some time
but I can remove for now